src/graphnet/models/standard_model.py

```diff
@@ -61,6 +61,15 @@ def __init__(
         self._scheduler_class = scheduler_class
         self._scheduler_kwargs = scheduler_kwargs or dict()
         self._scheduler_config = scheduler_config or dict()
+        self._configure_model_constants()
+
+    def _configure_model_constants(self) -> None:
+        self.target = [
+            target for task in self._tasks for target in task._target_labels
+        ]
+        self.prediction_columns = [
+            column for task in self._tasks for column in task._output_labels
+        ]
```

Comment:
Just a suggestion: Standardise the names of `target` -> `target_labels` and `preduction_columns` -> `prediction_labels`, and expose these as properties (e.g., `model.target_labels`).
```suggestion

    @property
    def target_labels(self) -> List[str]:
        return [
            label for task in self._tasks for label in task._target_labels
        ]
    
    @property
    def prediction_labels(self) -> List[str]:
        return [
            label for task in self._tasks for label in task._output_labels
        ]
```